### PR TITLE
UI: Select business service for domain

### PR DESCRIPTION
**Pull request by OferLevi85**

### CODE REVIEW

File: ui/src/components/modal/BusinessServiceModal.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright 2020 Verizon Media
```
Comment (abvaidya) [inline ui/src/components/modal/BusinessServiceModal.js:2]: copyright should be "The Athenz authors"

File: ui/src/components/modal/BusinessServiceModal.js [hunk 2]
```diff
@@ -0,0 +1,186 @@
+/*
```
Comment (abvaidya) [inline ui/src/components/modal/BusinessServiceModal.js:1]: Do we need a separate modal for businessservice and we cant use the existing modal?

File: ui/src/components/domain/ManageDomains.js [hunk 3]
```diff
@@ -22,6 +22,8 @@ import DeleteModal from '../modal/DeleteModal';
 import Color from '../denali/Color';
 import DateUtils from '../utils/DateUtils';
 import RequestUtils from '../utils/RequestUtils';
+import Link from 'next/link';
```
Comment (pratikgote) [inline ui/src/components/domain/ManageDomains.js:25]: please remove unused import

**Review by pratikgote — CHANGES_REQUESTED** [review]

**Review by pratikgote — APPROVED** [review]
👍

**Pull request merged**